deposit/src/main/java/edu/unc/lib/deposit/work/DepositSupervisor.java

```diff
@@ -57,13 +57,14 @@
 import edu.unc.lib.deposit.validate.ExtractTechnicalMetadataJob;
 import edu.unc.lib.deposit.validate.PackageIntegrityCheckJob;
 import edu.unc.lib.deposit.validate.ValidateContentModelJob;
-import edu.unc.lib.deposit.validate.ValidateFileAvailabilityJob;
 import edu.unc.lib.deposit.validate.ValidateDescriptionJob;
+import edu.unc.lib.deposit.validate.ValidateFileAvailabilityJob;
```

Comment:
Sorry to be predictable, but could you undo the changes to this file other than the actual new functionality? Eclipse went pretty egregious on that formatting so i'm having a hard time picking out what actually changed.
Might be repeating what you already know, so feel free to ignore, but in eclipse, if you right click and go to "Compare With" > "Previous Revision" it should give you the diff, and then you can use the buttons at the top of the diff to copy over the previously versions of chunks of the file (github client probably offers similar)